src/NumericalAlgorithms/LinearSolver/Convergence.hpp

```diff
@@ -0,0 +1,169 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines DataBox tags for the linear solver
+
+#pragma once
+
+#include <boost/none.hpp>
+#include <boost/optional.hpp>
+#include <cstddef>
+#include <iosfwd>
+
+#include "NumericalAlgorithms/LinearSolver/IterationId.hpp"
+#include "Options/Options.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+namespace PUP {
+class er;
+}  // namespace PUP
+/// \endcond
+
+namespace LinearSolver {
+
+/*!
+ * \brief Criteria that determine the linear solve has converged
+ *
+ * \details The following criteria are implemented:
+ *
+ * - MaxIterations: Matches if the number of iterations exceeds this limit.
+ * - AbsoluteResidual: Matches if the residual has reached this magnitude.
+ * Bounded below by the product of the machine epsilon and the conditioning
+ * number of the linear operator that is being inverted.
```

Comment:
I don't understand what the second sentence ("Bounded below...") is trying to tell me.

Reply:
I added more documentation. Does this make it more clear?